src/components/util/forms/contacts/add-contact.jsx

```diff
@@ -0,0 +1,150 @@
+import { React, useEffect, useState } from "react";
+import PropTypes from "prop-types";
+import set from "lodash.set";
+import { Button, Col, Row } from "react-bootstrap";
+import { ulid } from "ulid";
+import { useTranslation } from "react-i18next";
+import MediaSelector from "../../../slide/content/media-selector";
+import FormInput from "../form-input";
+
+/**
+ * An input for adding contacts.
+ *
+ * @param {string} props The props.
+ * @param {Function} props.onMediaChange For keeping images updated
+ * @param {Function} props.addContact For adding contact
+ * @param {Function} props.contact The contact currently being edited
+ * @param {Function} props.getInputImage For getting contact image
+ * @returns {object} Add contact component.
+ */
+function AddContact({
```

Comment:
This component is used for both add and editing. So it should have another name that reflects this.

Reply:
As well as the description.

Reply:
renamed to AddEditContact